crates/daphne-service-utils/src/config.rs

```diff
@@ -22,17 +24,36 @@ pub struct TaskprovConfig {
     #[serde(with = "hex")]
     pub vdaf_verify_key_init: [u8; 32],
 
-    /// Leader, Helper: Method for authorizing Leader requests.
-    #[serde(with = "from_raw_string")]
-    pub leader_auth: DaphneWorkerAuthMethod,
+    /// Peer authentication tokens.
+    pub peer_auth: PeerAuth,
```

Comment:
```suggestion
    pub peer_bearer_token: PeerAuth,
```

Reply:
this change makes configuration go from
```
peer_auth.leader.expected_token = "I-am-the-leader" # SECRET
```
to
```
peer_bearer_token.leader.expected_token = "I-am-the-leader" # SECRET
```

which is a bit redundant. I can instead change the `PeerAuth` type to `PeerBearerToken`

Reply:
changing the name of the type SGTM